src/NServiceBus.Core/Extensibility/ReadonlyContextBag.cs

```diff
@@ -8,11 +8,19 @@ public interface ReadOnlyContextBag
         /// <summary>
         /// Retrieves the specified type from the context.
         /// </summary>
-        /// <typeparam name="T">The type to retrieve.</typeparam>
+        /// <typeparam name="T">The type to retrieve. The Fullname of will be used to look up the instance.</typeparam>
```

Comment:
I'd go with: "The type to retrieve. The fully qualified name of the type will be used to look up the instance."